tests/e2e/ctl_v3_member_test.go

```diff
@@ -62,31 +50,9 @@ func TestCtlV3MemberRemoveClientAutoTLS(t *testing.T) {
 func TestCtlV3MemberRemovePeerTLS(t *testing.T) {
 	testCtl(t, memberRemoveTest, withQuorum(), withNoStrictReconfig(), withCfg(*e2e.NewConfigPeerTLS()))
 }
```

Comment:
These functions are related to `quorum` and `noStrictReconfig`. It looks like they should be tested on e2e. I left them as they are.